test/upload_validation_test.ts

```diff
@@ -849,6 +862,41 @@ describe('Pack metadata Validation', () => {
         assert.isTrue(childSchema.required);
       });
 
+      it('invalid sync table name', async () => {
+        const syncTable = makeSyncTable({
+          name: 'Sync@Table',
```

Comment:
spaces (only) are blocked in `makeSyncTable`, so it's less code to test a different char